helm/secrets/secret-notifier_mail.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2022 Broadcom. All rights reserved.
+# The term "Broadcom" refers solely to the Broadcom Inc. corporate affiliate
+# that owns the software below.
+# This work is licensed under the OpenAFC Project License, a copy of which is
+# included with this software program.
+
+# Secret used to send mail notification from AFC cluster via SMTP
+
+apiVersion: v1
+kind: Secret
+metadata:
+    name: notifier-mail
+stringData:
+    NOTIFIER_MAIL.json: |
+        {
+            "SERVER": "smtp.gmail.com",
+            "PORT": 465,
+            "USE_TLS": false,
+            "USE_SSL": true,
+            "USERNAME": "afc.admin@broadcom.com",
+            "PASSWORD": "lfgerlpiowywytpi"
```

Comment:
Why?

Reply:
Oops, that was a mistake!

Reply:
Corrected.
However frankly I do not have a clear idea how secrets to be handled - this is close to top of my tofo stack